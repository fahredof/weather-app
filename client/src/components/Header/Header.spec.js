import React from 'react';
import Header from "./Header";
import icon from "./svg/icon.svg";

describe(`Header image`, () => {
    const header = shallow(<Header/>);

    it(`Содержит класс: header`, () => {
        expect(header.hasClass(`header`)).toBeTruthy();
    });

    it(`Рендеринг картинки`, () => {
        expect(header.find("img").prop("src")).toEqual(icon);
    });

    it(`отображается корректно`, () => {
        expect(shallowToJson(header)).toMatchSnapshot();
    });

});