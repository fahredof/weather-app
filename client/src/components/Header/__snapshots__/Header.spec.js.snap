// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header image отображается корректно 1`] = `
<div
  className="header"
>
  <form
    onClick={[Function]}
  >
    <img
      alt="icon"
      src="icon.svg"
    />
  </form>
</div>
`;
